privacy-protections/storage-partitioning/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Storage Partitioning Test Page</title>
+
+    <script src='/helpers/idb-wrapper.js' defer></script>
+    <script src='/helpers/uuidv4.min.js'></script>
+    <script src='./helpers/tests.js'></script>
+    <script src='./helpers/common.js'></script>
+    <script src='./main.js' defer></script>
+    <link href='./style.css' rel='stylesheet'></link>
+    <link id="favicon" rel="shortcut icon" type="image/png" href="" />
+</head>
+<body>
+    <p><a href="../../">[Home]</a> ↣ <a href="../">[Privacy Protections Tests]</a> ↣ <strong>[Storage Partitioning Test Page]</strong></p>
+
+    <p>This page will determine whether various storage APIs are parititoned or blocked across origins.</p>
+    <p>NOTE: These tests will return incorrect results if:
+        <ul>
+            <li>you have multiple copies of the test page open</li>
+            <li>you run the tests after force reloading the page, as this <a href="https://web.dev/service-worker-lifecycle/#shift-reload">bypasses</a> Service Workers</li>
```

Comment:
I also get different results with devtools open in Chrome, so we can note that if you can reproduce